geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisHash.java

```diff
@@ -145,13 +148,15 @@ private void shutDownHscanSnapshotScheduledRemoval() {
   public synchronized void toData(DataOutput out, SerializationContext context) throws IOException {
     super.toData(out, context);
     DataSerializer.writeHashMap(hash, out);
+    DataSerializer.writeInteger(hashSize.get(), out);
```

Comment:
I wonder if we actually want to serialize this field or just recompute it during deserialization?

Reply:
from discussion with @ringles: it seems like writing it over the wire would save us some time recomputing the value. as the data set gets bigger, that could take us a good chunk of time. Is there a disadvantage to serializing it that we are not aware of?